### ISSUE TITLE
Incorrect conversion rate of RF

### ISSUE THREAD

**Issue by alan4678** [covers1624/PowerConverters#33]
Originally the design of conversion rates of different energy are based on the cost to smelt a single item in a furnace which proposed by Power Crystals in 
http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/1287424-1-5-1-1-5-2-powercrystals-mods-the-updates-never

Since he was calculated that at old version of thermal expansion, i.e. TE2 with 160 MJ / item (http://ftbwiki.org/Powered_Furnace), the new energy usage in TE3 and TE4 at 1.7.10 is 1600RF / item (http://ftbwiki.org/Redstone_Furnace). Hence, it should be fixed such that the ratio of EU:RF should be 1:4, which is the universal ratio and already applied on other mods such as MFR and AE2.

For converting steam, as a Steam Oven can smelt 9 items for every 8000mb of steam (http://ftbwiki.org/Steam_Oven), the average value is 888mb / item. The ratio of steam should also be adjusted. Note: 40mb of steam will produce 80RF by using Steam Dynamo (http://ftbwiki.org/Steam_Dynamo).

Since Factorization and PneumaticCraft does not have a complete wiki page for reference, so this two mods are open for any reasonable ratio.

In short, I hope that the ratio can be adjusted since I have already built lots of High Voltage Solar Array but does not have a better way to convert it back to RF. Power Converters is the best and most convenient mod that help my world constructions a lot and I hope it can keep going for better.

Code that affect the ratio of conversion:
https://github.com/covers1624/PowerConverters/blob/master/src/main/java/covers1624/powerconverters/init/PowerSystems.java#L24

@covers1624

**Comment by covers1624** [comment]
Roger, ill look in to it and do a final 1.7 release soon.

**Comment by Speiger** [comment]
@alan4678 EU To RF Convertion is wrong. Because they did not take time into account. Best example:
A RF Generator from TE3 produces its power quicker as EU & MJ... So his power ratio is very unbalanced.... Thats why the ratio should be at least 10-1 (10RF = 1EU)

**Comment by covers1624** [comment]
@Speiger No. The ratio will be 4:1. It seems stupid to have it 10:1 as all the other mods have based themselves on 4:1. Also MJ is not a thing anymore and the ratio was changed in the last commit.

**Comment by Speiger** [comment]
@covers1624 yeah RF is broken in time based power production even if the piece of coal is the same power when you use them on both. But in TE3s generators its generating 2-2.5x faster which increases the power production per tick drasticly.

**Comment by alan4678** [comment]
@Speiger If you only consider the generation rate of EU compare to RF, of course the EU need to take longer due to its property. But look carefully, 1 coal will produce 10s*20t/s*80RF/t=16000RF with Steam Dynamo and 4000EU with the Generator in IC2 (http://wiki.industrial-craft.net/index.php?title=Generator). The ratio is clearly 4:1. If you think that producing EU is too slow, how about converting RF back to EU with the coming new version of Power Converter? You should be noted that the conversion is bidirectional, every disadvantages you have is the advantages of the reverse process.

**Comment by Speiger** [comment]
i solved it with a 2 sided convertion config... Which had a 2 sided disadvantage (40 RF = 1EU & 1 EU = 4 RF) with that its not unbalanced and you still gain enough energy... Its still better then a 1 Side config which can be an exploit for either side... (BR & other mods who have stupid amounts of gen)

**Comment by Speiger** [comment]
@alan4678 they accept it because its a config and they can change it. Loops are caught by the config.

**Comment by Speiger** [comment]
@covers1624 @alan4678 i think this thing shows that RF itself is unbalanced and EU to RF is not 4 to 1 or back... https://www.reddit.com/r/feedthebeast/comments/4pak8h/i_compared_all_coal_generators/
